packages/navigation/src/styles/Navigation.module.css

```diff
@@ -176,3 +189,7 @@
 .NavItem-tooltip {
   text-align: center;
 }
+
```

Comment:
You could try targeting `.NavigationMenuItem svg` and see if that works, otherwise this seems reasonable. We could also modify the API so that `icon` takes a ReactNode, which you can pass a class to. Then, internally in MenuItem we'd clone the element and add the `Menu-item-icon` class as well.